tests/Http/Offer/UpdatePriceInfoRequestHandlerTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Offer;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Event\Commands\UpdatePriceInfo as EventUpdatePriceInfo;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\ApiProblem\SchemaError;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Place\Commands\UpdatePriceInfo as PlaceUpdatePriceInfo;
+use CultuurNet\UDB3\PriceInfo\BasePrice;
+use CultuurNet\UDB3\PriceInfo\PriceInfo;
+use CultuurNet\UDB3\PriceInfo\Tariff;
+use CultuurNet\UDB3\StringLiteral;
+use CultuurNet\UDB3\ValueObject\MultilingualString;
+use Money\Currency;
+use Money\Money;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class UpdatePriceInfoRequestHandlerTest extends TestCase
```

Comment:
This file is missing a lot of tests to validate the price info. All the validation rules from `PriceInfoDataValidator` that are validated here `PriceInfoDataValidatorTest` should now be verified here.

A good example to see tests for the validation of new request handlers is `UpdateCalendarRequestHandlerTest`

When the endpoint is using the new approach `PriceInfoDataValidator` can even be removed.